models/RoutesParser.cfc

```diff
@@ -158,6 +158,17 @@ component accessors="true" threadsafe singleton {
 			}
 		}
 
+		// Remove any route excludes by prefix
+		if ( !!arrayLen( moduleSettings.excludeRoutesPrefix ) ) {
```

Comment:
No need to double bang this.  You can use straight-up boolean evaluation, per Ortus coding standards